pdf-tools fails to compile on 10.4.1

Running 10.4.1, I get:

```
$ brew install pdf-tools --HEAD
Cannot read termcap database;
using dumb terminal settings.
==> Installing pdf-tools from dunn/emacs
Cannot read termcap database;
using dumb terminal settings.
==> Cloning https://github.com/politza/pdf-tools.git
Updating /Users/benwibking/Library/Caches/Homebrew/pdf-tools--git
==> Checking out branch master
Already on 'master'
Your branch is up to date with 'origin/master'.
HEAD is now at b6c0e4f Fix indentation
==> make server/epdfinfo
Last 15 lines from /Users/benwibking/Library/Logs/Homebrew/pdf-tools/01.make:
  "_synctex_node_tag", referenced from:
      _cmd_synctex_backward_search in epdfinfo-epdfinfo.o
  "_synctex_scanner_free", referenced from:
      _cmd_synctex_forward_search in epdfinfo-epdfinfo.o
      _cmd_synctex_backward_search in epdfinfo-epdfinfo.o
  "_synctex_scanner_get_name", referenced from:
      _cmd_synctex_backward_search in epdfinfo-epdfinfo.o
  "_synctex_scanner_new_with_output_file", referenced from:
      _cmd_synctex_forward_search in epdfinfo-epdfinfo.o
      _cmd_synctex_backward_search in epdfinfo-epdfinfo.o
ld: symbol(s) not found for architecture x86_64
clang: error: linker command failed with exit code 1 (use -v to see invocation)
make[2]: *** [epdfinfo] Error 1
make[1]: *** [all] Error 2
make: *** [server/epdfinfo] Error 2

If reporting this issue please do so at (not Homebrew/brew or Homebrew/core):
https://github.com/dunn/homebrew-emacs/issues

```

--- a/Formula/pdf-tools.rb
+++ b/Formula/pdf-tools.rb
@@ -20,7 +20,7 @@ class PdfTools < EmacsFormula
   depends_on "dunn/emacs/tablist"
 
   def install
-    system "make", "server/epdfinfo"
+    system "make", "server/epdfinfo", "CC=/usr/bin/gcc", "CXX=/usr/bin/gcc", "AR=/usr/bin/ar", "RANLIB=/usr/bin/ranlib"
     bin.install "server/epdfinfo"
 
     byte_compile Dir["lisp/pdf*"]

